csrc/expr_simplifier.cpp

```diff
@@ -992,16 +991,14 @@ Val* divideFactorized(Val* x, Val* y) {
   auto x_factors = getConstAndSymbolicFactors(x);
   auto y_factors = getConstAndSymbolicFactors(y);
 
-  int64_t quoient_const_factor;
```

Comment:
Error (CLANGTIDY) [cppcoreguidelines-init-variables,-warnings-as-errors] variable 'quoient_const_factor' is not initialized

         992  |  auto x_factors = getConstAndSymbolicFactors(x);
         993  |  auto y_factors = getConstAndSymbolicFactors(y);
         994  |
    >>>  995  |  int64_t quoient_const_factor;
         996  |  std::vector<Val*> quoient_symbolic_factors;
         997  |
         998  |  if (*x_factors.first->getInt() % *y_factors.first->getInt() != 0) {